src/seedu/addressbook/storage/Storage.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.storage;
+
+import seedu.addressbook.data.AddressBook;
+
+/**
+ * Created by Francis on 21/9/2017.
+ */
+public abstract class Storage {
+
+    abstract public AddressBook load() throws Exception;
+    abstract public void save(AddressBook addressBook) throws Exception;
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.